package.json

```diff
@@ -10,11 +10,11 @@
     "build": "concurrently 'npm:js' 'npm:css'",
     "build:dev": "concurrently 'npm:js:dev' 'npm:css:dev'",
     "build:watch": "concurrently 'npm:js:watch' 'npm:css:watch'",
-    "js": "NODE_ENV=production webpack --mode production",
-    "js:dev": "NODE_ENV=development webpack --mode development",
-    "js:watch": "NODE_ENV=development webpack --mode development --watch",
-    "css": "NODE_ENV=production postcss src/main/css/{style,login}.css --dir target/classes/static/css",
-    "css:dev": "NODE_ENV=development postcss src/main/css/{style,login}.css --dir target/classes/static/css",
+    "js": "npx cross-env NODE_ENV=production webpack --mode production",
+    "js:dev": "npx cross-env NODE_ENV=development webpack --mode development",
+    "js:watch": "npx cross-env NODE_ENV=development webpack --mode development --watch",
+    "css": "npx cross-env NODE_ENV=production postcss src/main/css/{style,login}.css --dir target/classes/static/css",
+    "css:dev": "npx cross-env NODE_ENV=development postcss src/main/css/{style,login}.css --dir target/classes/static/css",
```

Comment:
Hat das einen speziellen Grund für `npx` vs in der `package.json` `dependencies` Liste mit aufzunehmen?

Reply:
Vermutlich ist der Hauptgrund meine Unerfahrenheit in Sachen Node.js. In verschiedenen Lösungen hatte ich den Aufruf durch npx gefunden. Nachdem ich jetzt nochmal nachrecherchiert habe denke ich aber Du hast Recht und eine dependency ist die bessere Lösung. Ich schiebe noch einen Commit hinterher, der cross-env in die devDependencies aufnimmt und auf npx verzichtet.

Danke für den kritischen Hinweis.

Reply:
Commit ist drin, bitte um nochmaliges Review. Thx!